routing/services/routing.yml

```diff
@@ -0,0 +1,72 @@
+services:
```

Comment:
It's very confusing to have services defined in a file named `routing/services/routing.yml`.

Also, is `.yml` the proper extension or should it be `.yaml`?

Reply:
It can be both, but all applications I know use .yml... We used both at some point, so I picked one for consistency..
You have a valid point about the confusion, but I would have to figure out what the consequences are if we rename it

Reply:
> I'm also particularly interested in replacing the hooks functionality, and if I recall correctly from previous conversations, this would be possible with services. How could we use that with this setup? Would it be possible to have something analogous to what we have today, where we call a hook, and then all hooks defined by enabled modules are executed? Asking for a friend rofl
> 
> @sgomez maybe you can help us out a bit? @bendikrb do you have any suggestions?

I would recommend to implement EventDispatcher, and have the modules subscribe to the various events (i.e. "hooks").
I'm looking in to how to best do this now - while also keeping it somewhat backwards compatible. This ties together with having a service container at the core too - as far as I can grasp.